regions_www/m/pct_shiny_funs.R

```diff
@@ -30,11 +30,11 @@ normalise <- function(values, min = 0, max = 1){
 }
 
 
-## Create percentages such that rounded to XDP if >0% and <0.5%.  [NB can be NaN if denominator zero]
+## Create Values such that rounded to XDP if >0 and <0.5.  [NB can be NaN if denominator zero]
 round_dp <- function(expression, small_threshold = 0.05, large_threshold = 0.5){
-  small <- is.finite(expression) & expression > 0 & expression < small_threshold
-  medium <- is.finite(expression) & expression >= small_threshold & expression < large_threshold
-  large <- is.finite(expression) & expression >= large_threshold
+  small <- is.finite(expression) & expression > (small_threshold*-1) & expression < small_threshold
+  medium <- is.finite(expression) & ((expression <= (small_threshold*-1) & expression > (large_threshold*-1)) | (expression >= small_threshold & expression < large_threshold))
+  large <- is.finite(expression) & ((expression <= (large_threshold*-1)) | (expression >= large_threshold))
```

Comment:
:clap: nice. 
I don't think you need to `expression > (small_threshold*-1)`, you can just do `expression > -small_threshold`

Reply:
Ah yes, so you can :) have done that